app/src/components/ChoiceCard.tsx

```diff
@@ -140,7 +136,7 @@ function ChoiceCard({
   return (
     <div
       role="region"
-      className="card h-100 border-0 bg-transparent"
```

Comment:
We do have a card class in tailwind, do you think we should use it here? @divyaYK 
what do you think? @tiagostutz